### ISSUE TITLE
[虫子]：It seems that this project cannot run for a long time in the docker of the R2S soft route.

### ISSUE THREAD

**Issue by Whys8182** [navidrome/navidrome#2348]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

上一个

### Current Behavior

It seems that this project cannot run for a long time in the docker of the R2S soft route. At present, I only store songs within 20GB in it. Various abnormalities always occur in the docker of this device, and it cannot always be started, restarted, or stopped. Can't run normally for a long time!

### Expected Behavior

Can't run normally for a long time!

### Steps To Reproduce

Can't run normally for a long time!

### Environment

```markdown
- OS:
- Browser:
- Client:
- Openwrt：OpenWrt 04.18.2023 by Kiddin' / LuCI Master git-107d977
（Firmware download address: https://supes.top/releases/targets/rockchip/armv8/openwrt-04.18.2023-rockchip-armv8-friendlyarm_nanopi-r2s-squashfs-sysupgrade.img.gz）
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
Sorry, but I don't have access to this hardware to properly test this. Do you have issues with other docker containers or is it just Navidrome? You can try to disable periodic scanning, by setting ND_SCANSCHEDULE=0.

**Comment by mybuntu-git2** [comment]
This workaround works and we can wait the next version to test automatic scanning. Please see https://github.com/navidrome/navidrome/issues/2349
for a docker compose example.

**Comment by mybuntu-git2** [comment]
@deluan I see there is a lot of other work going on, was wondering if the fix for ARM ND_SCANSCHEDULE=0 is in the pipeline. Right now at container start the memory over allocation does not happen and new items are scanned correctly. The issue is there only for scans when the container is already running.

**Comment by tty228** [comment]
Perhaps you can try using the binary file directly. this is an OpenWrt LuCI management interface:
https://github.com/tty228/luci-app-navidrome/blob/main/README_en.md

**Comment by mybuntu-git2** [comment]
In relation to memory overallocation as per #2349 I confirm the issue to be permanently resolved in 0.50.1 with additionally a decrease in memory usage on ARM by 50% or more

**Comment by deluan** [comment]
Hey @Whys8182 can you confirm this issue has been resolved in the latest version, 0.50.2?
